src/container.js

```diff
@@ -106,12 +106,24 @@ class ScrollMonitorContainer {
 	}
 
 	listenToDOM () {
+		var browserSupportsPassive = false;
+		try {
+			var opts = Object.defineProperty({}, 'passive', {
+				get: function() {
+					browserSupportsPassive = true;
+				}
+			});
+			window.addEventListener('test', null, opts);
+		} catch (e) {}
+
+		const useCapture = browserSupportsPassive ? {capture: false, passive: true} : false;
+
```

Comment:
We're almost there. Move this block up to the top level of the module, right now it's calling `addEventListener` every time a watcher is created.